Cannot find module 'graceful-fs'

I am getting this error all other task are working fine accept tinypng I am using windows 8.1 npm 0.12 and grunt 0.4+

Loading "tinypng.js" tasks...ERROR
>> Error: Cannot find module 'graceful-fs'
Warning: Task "tinypng" not found. Use --force to continue.

Aborted due to warnings.

--- a/package.json
+++ b/package.json
@@ -25,7 +25,6 @@
     "test": "grunt test"
   },
   "devDependencies": {
-    "graceful-fs": "^3.0.6",
     "grunt": "~0.4.1",
     "grunt-contrib-clean": "~0.4.0",
     "grunt-contrib-jshint": "~0.6.0",
@@ -44,6 +43,7 @@
     "test": "test"
   },
   "dependencies": {
+    "graceful-fs": "^3.0.6",
     "grunt": "~0.4.1",
     "grunt-contrib-clean": "~0.4.1",
     "grunt-contrib-jshint": "~0.6.5",

